transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpResponseDecodingState.java

```diff
@@ -257,7 +257,8 @@ protected DecodingState finishDecode(List<Object> childProducts,
 						// create the cookie name and value
 						DefaultHttpCookie cookie = new DefaultHttpCookie(
 								nvPair[0].trim());
-						cookie.setValue(nvPair[1].trim());
+						if (nvPair.length > 1)
+						    cookie.setValue(nvPair[1].trim());
```

Comment:
@msalavastru This fix seems fine. Where's the k3po test you mentioned that verifies this fix? Also, as mentioned in our call just now, please add a test method to unit test HttpResponseDecoderTest to confirm the fix (make sure it fails without the fix).